packages/core/src/extensions/TableHandles/TableHandlesPlugin.ts

```diff
@@ -145,6 +146,16 @@ export class TableHandlesView<
       return;
     }
 
+    // Prevents table handles from showing while a selection is active.
+    if (!this.pmView.state.selection.empty) {
```

Comment:
won't this also hide the draghandles when the mouse is not down? i.e. when the user just selected a word in a table cell?